_beats/testing/environments/latest.yml

```diff
@@ -3,7 +3,7 @@
 version: '2.1'
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:6.0.0-rc1
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.0.0-rc2
```

Comment:
Shouldn't this be `6.0.0` now?

Reply:
Good find: https://github.com/elastic/beats/pull/5648